app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/components/MainScaffold.kt

```diff
@@ -67,7 +67,7 @@ private fun MainScaffold(
             } else {
                 Column {
                     Box(modifier = Modifier.weight(1.0f)) { content() }
-                    HorizontalDivider()
+                    if (navType == NavigationSuiteType.NavigationBar) HorizontalDivider()
```

Comment:
If we don't show the HorizontalDivider() enter the first `if` case with a simple `content()`